features/file_sourcing.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Sourcing tcl files
+
+  Scenario: Display stack trace when there is an error in a sourced file
+    Given a file named "features/test.feature" with:
+      """
+      Feature:
+        Scenario:
+          Given testing file sourcing
+      """
+    And a file named "features/support/env.rb" with:
+      """
+      require 'cucumber/tcl'
+      """
+    And a file named "features/support/env.tcl" with:
```

Comment:
One comment on this. You used the name `env.tcl` for the file here. Did you know that the equivalent file in Ruby - `env.rb` has special status amongst all the files in the support folder, in that:

- it's loaded before all the others
- if you run Cucumber in `--dry-run` mode it's not loaded at all

This is intended to be the file where you boot up your system under test, so that the rest of your test code can load up on the assumption that the system is in that state.

I suggest using a different name for the file in this scenario, and adding a new ticket to have the same special behaviour for `env.tcl` as we do for `env.rb`. What do you think?